ui/app/AppLayouts/Chat/components/PrivateChatPopupSearchResults.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+import "./"
+
+
+Item {
+    id: root
+    height: 64
+    property bool hasExistingContacts: false
+    property bool showProfileNotFoundMessage: false
+    property bool loading: false
+    property string username: ""
+    property string userAlias: ""
+    property string pubKey: ""
+
+    signal resultClicked(string pubKey)
+    signal addToContactsButtonClicked(string pubKey)
+    anchors.left: parent.left
+    anchors.right: parent.right
```

Comment:
Pro tip: `width: parent.width` does the same as those `anchors` , but it saves one line! 😁